api/src/pcapi/routes/backoffice_v3/templates/pro_user/get/details/history.html

```diff
@@ -20,8 +20,15 @@
                 <th scope="row"></th>
                 <td>{{ action.actionType.value }}</td>
                 <td>{{ action.actionDate | format_date("Le %d/%m/%Y à %Hh%M") }}</td>
-                <td class="px-4">
+                <td class="px-4 text-break">
                     <p>{{ action.comment | empty_string_if_null }}</p>
+                    {% if action.actionType.name == "USER_INFO_MODIFIED" %}
+                        {% for modified_data in action.extraData['extra_data'] %}
```

Comment:
J'aurais quand même protégé au cas où les données n'existent pas
{% for modified_data in action.extraData.get('changes', {}) %}